rest_framework/routers.py

```diff
@@ -135,11 +135,7 @@ def get_default_base_name(self, viewset):
         If `base_name` is not specified, attempt to automatically determine
         it from the viewset.
         """
-        queryset = getattr(viewset, 'queryset', None)
-
-        assert queryset is not None, '`base_name` argument not specified, and could ' \
-            'not automatically determine the name from the viewset, as ' \
-            'it does not have a `.queryset` attribute.'
+        queryset = viewset().get_queryset()
```

Comment:
I was under the impression that all viewsets will have a get_queryset defined. Is that not true?